README.md

```diff
@@ -18,42 +18,34 @@ Then, select one of the following methods.
 
 ### Using Carthage
 
-Checkout the tag PolyNetSDK-version-5-carthage in the repository.
-
-```shell
-git checkout PolyNetSDK-version-5-carthage
-```
-
-Update the carthage dependencies for the specific Apple platform. For example, iOS for PolyNetSampleSwift.
+Update the carthage dependencies for the specific Apple platform. 
 
 ```shell
 carthage update --platform ios --use-xcframeworks
 ```
 
-Run the project from Xcode.
+Go to the Frameworks folder inside the Xcode project (.xcodeproj) and remove the PolyNetSDK, Starscream, and SwiftProtobuf frameworks that are marked in red and which not being found by Xcode. After removing them, right-click inside the frameworks folder and select Add files to PolyNetSampleSwift. Make sure copy items if needed, create groups and add to targets PolyNetSampleSwift is selected. Then choose the XCFrameworks (PolyNetSDK, Starscream and SwiftProtobuf) from the PolyNetSampleSwift/Carthage/Build folder. 
+
+Once added, run the project from Xcode.
```

Comment:
After updating the Carthage dependencies, follow these steps (PolyNetSampleSwift project as a example):
1. Open the Xcode project (.xcodeproj) and navigate to the Frameworks folder.
2. Right-click the Frameworks folder and select "Add files to PolyNetSampleSwift."
3. Ensure that "Copy items if needed," "Create groups," and "Add to targets PolyNetSampleSwift" are selected.
4. Choose the XCFrameworks (PolyNetSDK, Starscream, and SwiftProtobuf) from the PolyNetSampleSwift/Carthage/Build folder.
5. Go to the "General" tab of your project settings and ensure that the added frameworks are set to "Embed & Sign."

Reply:
Done